src/routes/api/chat/+server.ts

```diff
@@ -0,0 +1,64 @@
+import { CHAT_API_KEY } from '$env/static/private';
```

Comment:
Would you mind adding a comment with the file path for where this key should go?

Reply:
Yup, also for future reference it's defined [here](https://kit.svelte.dev/docs/modules#$env-static-private)